01/RitamDey/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    words = []
```

Comment:
no need to initialize first as long as you dont have any conditional statements or a try except block, here words gets overwritten or open() throws an error, so never will be empty (besides the document is empty but then the list comprehension will be empty)